File: algorithms/code/hackerrank/simple_array_sum/simple_array_sum.py
# https://www.hackerrank.com/challenges/simple-array-sum

import unittest


def simple_array_sum(ar: list[int]) -> int:
    return sum(ar)


class TestCode(unittest.TestCase):
    def test_example(self) -> None:
        self.assertEqual(31, simple_array_sum([1, 2, 3, 4, 10, 11]))

File: algorithms/code/hackerrank/simple_array_sum/simple_array_sum.rs
// https://www.hackerrank.com/challenges/simple-array-sum

pub fn simple_array_sum(ar: &[i32]) -> i32 {
    let mut sum = 0;
    for value in ar {
        sum += value;
    }
    sum
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test_example() {
        assert_eq!(31, simple_array_sum(&[1, 2, 3, 4, 10, 11]));
    }
}
